sdk/file_sdk/XMPFileFormat/xmp_helper.cpp

```diff
@@ -79,12 +123,51 @@ const vector<Tag> XMPHelper::GetTags(shared_ptr<IStream> fileStream) {
   catch(XMP_Error error)
   {
     file.CloseFile();
-    throw std::runtime_error(error.GetErrMsg());
+    throw Exception(error.GetErrMsg());

```

Comment:
what is the Exception class?

Reply:
no need for new operator?

Reply:
i don't know why i forget that and how its compiling 
Slava created a wrapper class

Reply:
Wrapper class for what? Why?

I checked, and there is no need for new.

Reply:
I don't know why he created it but maybe because you can't use std::exception in Linux with string message